### PR TITLE
Register old committees on Pittsburgh scraper

### PR DESCRIPTION
**Pull request by jtotoole**
This PR registers some committees that no longer exist but were associated with old bills; their omission was causing imports drawing on old legislative sessions to fail. 

Our current and former mayor were both council members before being elected, and I'm still having trouble disambiguating their council vs. mayoral terms (see more detail comment inline). Any suggestions on that would be welcome before this gets merged.

### CODE REVIEW

File: pittsburgh/__init__.py [hunk 1]
```diff
@@ -38,32 +38,107 @@ def get_organizations(self):
...
+        hearing_sched.add_source(pgh_legistar_url, note="web")
+        yield hearing_sched
+
         mayor = Organization(name="Mayor", classification="executive")
         mayor.add_post("Mayor", "Mayor", division_id="ocd-division/country:us/state:pa/place:pittsburgh")
         mayor.add_source("http://pittsburghpa.gov/mayor/index.html", note="web")
         yield mayor
 
-        # TODO: figure out disambiguation for councilman/mayor positions (using birth_date?)
+        # TODO: figure out disambiguation for councilman/mayor positions—using birth_date isn't working
```
Comment (jtotoole) [inline pittsburgh/__init__.py]: @fgregg and I [talked about](https://github.com/opencivicdata/scrapers-us-municipal/pull/278#discussion_r297815760) how to distinguish between the mayoral and council terms of Peduto and Ravenstahl when I made my last PR, but unfortunately the suggestion of adding `birth_date` didn't work—got the same error when running locally with this section uncommented:

```
pupa.exceptions.SameNameError: multiple people with same name "William Peduto" in Jurisdiction - must provide birth_date to disambiguate
```

Comment (fgregg) [reply pittsburgh/__init__.py]: did you add birth_date in both places you create a person?

Comment (jtotoole) [reply pittsburgh/__init__.py]: looks like the fix was actually to `add_term` for the same user ids rather than instantiate two different `Person`s

**Comment by jtotoole** [comment]
i'm actually gonna close this out for now as I've identified a couple other places where this scraper can be improved that will require a decent amount more work

**Pull request closed without merging**